test/index.test.ts

```diff
@@ -723,3 +723,24 @@ test('parse ps output', () => {
     ]
   })
 })
+
+test('returns progres of command before promise resolves', async (): Promise<void> => {
+  
+  let firstCallbackFire
+
+  const config = {
+    cwd: path.join(__dirname),
+    config: './docker-compose.yml',
+    callback: (chunk) => {
```

Comment:
This callback should properly define the types.

Reply:
done